### PR TITLE
{Doc} Fix Minor Typo in Sample Command

### PR DESCRIPTION
**Pull request by mindlessroman**
**Related command**
n/a

**Description**
Minor typo in a doc.

**Testing Guide**
n/a

**History Notes**

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Documentation

**Review by yonzhan — APPROVED** [review]

File: doc/extensions/authoring.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ We detect Python package version via `python setup.py --version`. Only when the
 
 The storage fields can be stored in your config file or as environment variables so you need not supply them every time. Once the publish command has been run (you must be logged in to the Azure CLI for it to succeed), you can open a PR that will contain your code changes and the index update. This used to be done in two steps.
 
-Once your extension is published, you can view it via `az extension list-avaliable -o table`.
+Once your extension is published, you can view it via `az extension list-available -o table`.
```
Comment (jiasli) [inline doc/extensions/authoring.md:54]: This typo was from https://github.com/Azure/azure-cli/pull/13960. It should definitely be found out by a text editor with spell check.

**Review by jiasli — APPROVED** [review]

File: doc/extensions/authoring.md (hunk 1 above)
Comment (mindlessroman) [reply doc/extensions/authoring.md:54]: I know sometimes spell checks aren't always caught if they're in code snippets, which may have been the case.

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**